src/palace.pyx

```diff
@@ -2322,7 +2322,7 @@ cdef class ChorusEffect(BaseEffect):
 
     Parameters
     ----------
-    waveform : int
+    waveform : str
     phase : int
     depth : float
     feedback : float
```

Comment:
Please add description for these values based on the docs, since we're polishing things here.